plugins/tileacq.py

```diff
@@ -288,11 +314,27 @@ def show_dlg(self):
         ss = self._get_live_streams()
         self.filename.value = self._get_new_filename()
 
+        # On SECOM allow to do fine alignment for each tile
+        if self._can_fine_align(ss):
+            self.vaconf["fine_align"] = {"label": "Fine alignment"}
```

Comment:
You can add this to the initial vaconf, no need to extend it on the fly.
However, we need a way to show/hide it whenever the window is shown. I think there are two ways: either create/delete the fine_align VA every time here, or you can tweak the .vaconf to change add a `"control_type": odemis.gui.CONTROL_NONE` when you don't want to show it. This second solution might be easier as the rest of the code won't have to be full of `if hasattr(self, "fine_align")....`.